setup.py

```diff
@@ -35,16 +35,16 @@ def run(self):
     'requests >= 2.20.0, < 2.21',
     'fstrings',
     'msrestazure~=0.4.32',
-    'azure-cli-core >= 2.0.0, <2.0.55',
+    'azure-cli-core >= 2.0.0, <=2.0.62',
```

Comment:
depends on a beta version of PyYAML starting version 2.0.55, which will conflict with docker-compose. See #373 
Is this  issue avoided?

Reply:
Yes
1. The latest azure-cli-core doesn't specify version for pyyaml now [https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/setup.py#L70](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/setup.py#L70)
2. docker-compose also bumped pyyaml version range to >= 3.10, < 4.3 [https://github.com/docker/compose/blob/master/setup.py#L35](https://github.com/docker/compose/blob/master/setup.py#L35)